website/content/docs/connect/config-entries/service-intentions.mdx

```diff
@@ -21,38 +21,44 @@ The following outline shows how to format the service intentions configuration e
 - [`Name`](#name): string | required  
 - [`Namespace`](#namespace): string |  `default` | <EnterpriseAlert inline/>
 - [`Partition`](#partition): string |  `default` | <EnterpriseAlert inline />
-- [`Meta`](#meta): map | no default 
-- [`Sources`](#sources): list | no default
-  - [`Name`](#sources-name): string | no default
-  - [`Peer`](#sources-peer): string | no default
-  - [`Namespace`](#sources-namespace): string | no default  | <EnterpriseAlert inline />
-  - [`Partition`](#sources-partition): string | no default  | <EnterpriseAlert inline />
-  - [`Action`](#sources-action): string | no default  | required for L4 intentions
-  - [`Permissions`](#sources-permissions): list | no default
-    - [`Action`](#sources-permissions-action): string | no default  | required
+- [`Meta`](#meta): map 
+- [`JWT`](#jwt): map 
+  - [`Providers`](#jwt-providers): list of maps
+    - [`Provider`](#jwt-providers-provider): string
+    - [`VerifyClaims`](#jwt-provider-verifyclaims): list of maps
+      - [`Path`](#jwt-provider-verifyclaims-path): list of strings
+      - [`Value`](#jwt-provider-verifyclaims-value): string
+- [`Sources`](#sources): list 
+  - [`Name`](#sources-name): string 
+  - [`Peer`](#sources-peer): string 
+  - [`Namespace`](#sources-namespace): string | <EnterpriseAlert inline />
+  - [`Partition`](#sources-partition): string | <EnterpriseAlert inline />
+  - [`Action`](#sources-action): string | required for L4 intentions
+  - [`Permissions`](#sources-permissions): list
+    - [`Action`](#sources-permissions-action): string | required
     - [`HTTP`](#sources-permissions-http): map | required
-      - [`PathExact`](#sources-permissions-http): string | no default 
-      - [`PathPrefix`](#sources-permissions-http): string | no default 
-      - [`PathRegex`](#sources-permissions-http): string | no default 
-      - [`Methods`](#sources-permissions-http): list | no default 
-      - [`Header`](#sources-permissions-http-header): list of maps |no default 
+      - [`PathExact`](#sources-permissions-http): string
+      - [`PathPrefix`](#sources-permissions-http): string
+      - [`PathRegex`](#sources-permissions-http): string
+      - [`Methods`](#sources-permissions-http): list | no default
```

Comment:
minor: The `| no default` was removed from other items but not from `Methods`. Double checking whether this should stay or be removed?